### PR TITLE
update according to issues by apache conculeagues

### PR DESCRIPTION
**Pull request by DaanHoogland**
Fixes apache/cloudstack#4752

Fixes apache/cloudstack#4753

### CODE REVIEW

**Review by GabrielBrascher — COMMENTED** [review]
This is a draft but it already LGTM. Is there any pending addition?

**Comment by DaanHoogland** [comment]
needs some procuder to get to the public site (described in the readme i think) I think @sebbASF had another comment and I wondered if it should go on here as well, hence draft

**Comment by DaanHoogland** [comment]
@GabrielBrascher , I think this is ready for final review

**Comment by DaanHoogland** [comment]
cc @sebbASF ^^

**Review by GabrielBrascher — APPROVED** [review]
Thanks, @DaanHoogland. LGTM

**Review by rhtyd — COMMENTED** [review]
LGTM

File: content/contribute.html [hunk 1]
```diff
@@ -145,7 +145,7 @@ <h1 id="indicators">Apache CloudStack: Getting Involved</h1>
 
 <p>Developers, of course, are also welcome. See the <a href="developers.html">Developer Resources</a> page!</p>
 
-<p>If you just want to report a bug, that's easily done in our <a href="https://issues.apache.org/jira/browse/CLOUDSTACK">Jira bug tracking tool</a>.</p>
+<p>If you just want to report a bug, that's easily done in our <a href="https://github.com/apache/cloudstack/issues">github project issues list</a>.(just click <a href="https://github.com/apache/cloudstack/issues/new/choose">new issue</a></p>
```
Comment (DaanHoogland) [inline content/contribute.html:148]: this was done in the wrong place, need to redo.

**Pull request merged**